api/middleware/roleAuthentication.js

```diff
@@ -1,15 +1,43 @@
-const { getRoleById } = require('../roles/rolesModel');
+/*
+  Roles:
+  1 - super_admin
+  2 - admin  
+  3 - instructor
+  4 - parent
+  5 - child
+*/
+// Only allows endpoint access for Super Admins (1), Admins (2), and Instructors (3)
+const roleAuthenticationInstructor = (req, res, next) => {
```

Comment:
These 3 functions are okay for now, but would benefit from a refactor (for readability/DRY purposes).